docs/server/source/data-models/index.rst

```diff
@@ -16,4 +16,4 @@ This section unpacks each one in turn.
    asset-model
    crypto-conditions
    block-model
-   vote-model
+   ../schema/vote
```

Comment:
This is the same as https://github.com/bigchaindb/bigchaindb/pull/865/files#diff-4098e6de490b39b8141bf1abd2b474b6R19 so you end up with the same documentation inside the `data-models` and the root of the server docs.

I prefer what you did with the transaction schema. The description page still exists as before but now the key names are links to the schema.